public/scripts/plugins/pcr/templates/pcr_form_view.hbs

```diff
@@ -61,7 +61,7 @@
   <div class="form-group">
     <label class="col-sm-6 control-label" for="newProduct_partType"><small>RDP Part type</small></label>
     <div class="col-sm-6">
-      {{{select availablePartTypes id="newProduct_partType" class="col-sm-6 form-control input-sm" selected=state.partType}}}
+      {{{select availablePartTypes id="newProduct_partType" class="col-sm-6 form-control input-sm" selected="" required=true }}}
```

Comment:
This will prevent someone from selecting a partType, changing view and then coming back to have the correct partType selected.  I think that the change you might need to make as at the model level... were we have an invalid model who validity is only checked once the "Create new RDP part" button is clicked.